modules/integration/tests-ui-integration/tests-ui/src/test/resources/files/portal.json

```diff
@@ -4,17 +4,25 @@
       "fs"
     ]
   },
+  "gadgetGeneration": {
+    "isCreateGadgetEnable": false,
+    "ignoreProviders": [
+      "rt"
+    ]
+  },
   "authentication": {
     "activeMethod": "basic",
     "methods": {
       "sso": {
         "attributes": {
           "issuer": "portal",
-          "identityProviderURL": "%https.ip%/samlsso",
-          "responseSigningEnabled": "false",
-          "acs": "%https.ip%/portal/acs",
+          "identityProviderURL": "https://localhost:9443/samlsso",
```

Comment:
I updated the portal.json in consistent with the portal.json with carbon-dashboards. This was changed with https://github.com/wso2/carbon-dashboards/commit/2fbe8838a04015b0584ac4ae470a7056c10d230f#diff-b1d0f8773ac16551d7910e5033f7bc0b